Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -145,25 +138,30 @@ public async Task<IReadOnlyList<PackageInfo>> GetAvailablePackages(CancellationT
                     }
                     catch (SerializationException ex)
                     {
-                        const string LogTemplate =
-                            "Failed to deserialize the plugin manifest retrieved from {PluginManifestUrl}. If you " +
-                            "have specified a custom plugin repository manifest URL with --plugin-manifest-url or " +
-                            PluginManifestUrlKey + ", please ensure that it is correct.";
-                        _logger.LogError(ex, LogTemplate, manifestUrl);
+                        _logger.LogError(ex, "Failed to deserialize the plugin manifest retrieved from {Manifest}", manifest);
                         throw;
                     }
                 }
             }
             catch (UriFormatException ex)
             {
-                const string LogTemplate =
-                    "The URL configured for the plugin repository manifest URL is not valid: {PluginManifestUrl}. " +
-                    "Please check the URL configured by --plugin-manifest-url or " + PluginManifestUrlKey;
-                _logger.LogError(ex, LogTemplate, manifestUrl);
+                _logger.LogError(ex, "The URL configured for the plugin repository manifest URL is not valid: {Manifest}", manifest);
                 throw;
             }
```

Comment:
```suggestion
            }
            catch (HttpException ex)
            {
                _logger.LogError(ex, "An error occurred while accessing the plugin manifest: {Manifest}", manifest);
                return Enumerable.Empty<PackageInfo>();
            }
```

to catch exceptions in request.